state-chain/runtime/src/lib.rs

```diff
@@ -1240,8 +1246,21 @@ type PalletMigrations = (
 	pallet_cf_cfe_interface::migrations::PalletMigration<Runtime>,
 );
 
-type MigrationsForV1_7 =
-	(migrations::housekeeping::Migration, migrations::reap_old_accounts::Migration);
+type MigrationsForV1_7 = (
+	migrations::housekeeping::Migration,
+	migrations::reap_old_accounts::Migration,
```

Comment:
Hm not sure why these were in the version-specific upgrade list...

Reply:
neither was I... I can pull them out